test/testatomic.c

```diff
@@ -726,6 +731,7 @@ int main(int argc, char *argv[])
     RunFIFOTest(SDL_FALSE);
 #endif
     RunFIFOTest(SDL_TRUE);
+    SDL_Quit();
```

Comment:
Shouldn't SDL_Quit() be the last thing to be called?

Reply:
It should, but we want to clean up all still=reachable SDL allocations before printing all remaining non cleaned-up memory.
`SDLTest_CommonQuit` does the same:
https://github.com/libsdl-org/SDL/blob/f4aca6866ca24a05bbfffbc848a362b484f7935a/src/test/SDL_test_common.c#L2496-L2497
https://github.com/libsdl-org/SDL/blob/f4aca6866ca24a05bbfffbc848a362b484f7935a/src/test/SDL_test_common.c#L148-L151

Reply:
Ah, okay.